Philips.CodeAnalysis.Common/DiagnosticIds.cs

```diff
@@ -116,5 +116,6 @@ public enum DiagnosticIds
 		ThrowInformationalExceptions = 2121,
 		AvoidExceptionsFromUnexpectedLocations = 2122,
 		PassSenderToEventHandler = 2123,
+		DocumentUnhandledExceptions = 2124,

```

Comment:
PH2122: Ideally the squiggle would be with the throw rather than the method name.

Reply:
I agree, fixed